test/web/mockHmac.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import crypto from 'isomorphic-webcrypto';
+import * as base64url from 'base64url-universal';
+const _secret = base64url.decode(
+  '49JUNpuy7808NoTTbB0q8rgRuPSMyeqSswCnWKr0MF4');
+
+export class MockHmac {
```

Comment:
There is not a DB module for this?  Surprised to see this sort of implementation as a test helper.

Reply:
Based on discussion on irc, we concluded we'll keep using this as it is.